### PR TITLE
Add session.reject and trusted_cluster.create events

### PR DESCRIPTION
**Pull request by kimlisa**
resolves https://github.com/gravitational/teleport/issues/4507
resolves https://github.com/gravitational/teleport/issues/4459

#### Description
Adds missing events: `session.rejected`, `trusted_cluster.create`, `trusted_cluster.delete`

#### Screenshot
storybook (i also test it out manually, that's how i got the json for storybook)
Screenshot from 2020-10-09 16-54-00

Screenshot from 2020-10-09 16-55-32

Screenshot from 2020-10-09 16-55-47

### CODE REVIEW

**Comment by webvictim** [comment]
I think this also needs to cover `trusted_cluster.delete` as per https://github.com/gravitational/teleport/issues/4307

File: packages/teleport/src/services/audit/makeEvent.ts [hunk 1]
```diff
@@ -409,6 +409,11 @@ export const formatters: Formatters = {
       return `User [${user}] has ended interactive session [${event.sid}] on node [${node}] `;
     },
   },
+  [CodeEnum.SESSION_REJECT]: {
+    desc: 'Session Rejected',
+    format: ({ user, login, server_id, reason }) =>
+      `User [${user}] was denied session for [${login}@${server_id}] because [${reason}]`,
```
Comment (alex-kovoy) [inline packages/teleport/src/services/audit/makeEvent.ts:415]: `was denied session for` -> `was denied access to`

File: packages/teleport/src/services/audit/makeEvent.ts [hunk 2]
```diff
@@ -525,6 +530,11 @@ export const formatters: Formatters = {
     desc: 'Trusted Cluster Token Created',
     format: ({ user }) => `User [${user}] has created a trusted cluster token`,
   },
+  [CodeEnum.TRUSTED_CLUSTER_CREATED]: {
+    desc: 'Trusted Cluster Created',
+    format: ({ user, name }) =>
+      `User [${user}] has created a trusted cluster relationship with [${name}]`,
```
Comment (alex-kovoy) [inline packages/teleport/src/services/audit/makeEvent.ts:536]: `User [${user}] has created a trusted relationship with cluster [${name}]`

**Review by alex-kovoy — APPROVED** [review]
with some comments

File: packages/teleport/src/services/audit/makeEvent.ts (hunk 1 above)
Comment (kimlisa) [reply packages/teleport/src/services/audit/makeEvent.ts:415]: revised

File: packages/teleport/src/services/audit/makeEvent.ts (hunk 2 above)
Comment (kimlisa) [reply packages/teleport/src/services/audit/makeEvent.ts:536]: revised

**Comment by kimlisa** [comment]
> I think this also needs to cover `trusted_cluster.delete` as per [gravitational/teleport#4307](https://github.com/gravitational/teleport/issues/4307)

added
Screenshot from 2020-10-12 09-51-02
Screenshot from 2020-10-12 09-51-09

**Pull request merged**